.SRCINFO

```diff
@@ -1,19 +1,19 @@
 pkgbase = dracut-hook
-	pkgdesc = dracut hook for archlinux
-	pkgver = 0.3
-	pkgrel = 2
-	url = https://tackitt.net
+	pkgdesc = Install/remove hooks for dracut
+	pkgver = 0.4
+	pkgrel = 1
+	url = https://dracut.wiki.kernel.org/index.php/Main_Page
 	arch = any
 	license = BSD
 	depends = dracut
 	source = dracut-install
 	source = dracut-remove
 	source = 90-dracut-install.hook
-	source = 90-dracut-remove.hook
-	md5sums = ca2782fe7bd5e5ec01ba1043df8db092
-	md5sums = d47474311fe65faf9358727f75526389
-	md5sums = 8fa0ed0a9aa50f2ac6525d3264f04d76
-	md5sums = 679dbd817343d459252225000d71ec52
+	source = 60-dracut-remove.hook
```

Comment:
Why lower the priority? Shouldn't this be late so any hooks modifying modules (dkms, etc) are built and included?

Reply:
Since the kernel is already going to be deleted, it will avoid conflicts with many other tools that may detect `/boot/{vmlinuz, initramfs}` just like `mkinitcpio` remove hook does (it also starts with 60)